Cargo.toml

```diff
@@ -27,11 +27,11 @@ chain-time = { git = "https://github.com/input-output-hk/chain-libs.git", branch
 chain-impl-mockchain = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "master" }
 time = { version = "0.3", features = ["formatting", "parsing", "macros"] }
 itertools = "0.10"
-jcli = { git = "https://github.com/input-output-hk/jormungandr.git", branch = "master" }
-jormungandr-lib = { git = "https://github.com/input-output-hk/jormungandr.git", branch = "master" }
-jormungandr-integration-tests = { git = "https://github.com/input-output-hk/jormungandr.git", branch = "master" }
-jormungandr-automation = { git = "https://github.com/input-output-hk/jormungandr.git",  branch = "master" }
-thor = { git = "https://github.com/input-output-hk/jormungandr.git",  branch = "master" }
+jcli = { git = "https://github.com/input-output-hk/jormungandr.git", branch = "update_jortestkit" }
```

Comment:
is this needed?

Reply:
it was needed, because in vit-testing we got a conflict (jormungandr has to be adjust before updating jortestkit), since now this branch is merged, we can discard this change

Reply:
This might be a problem for the fund8 release, since we will have to rely on the jormungandr fund8 branch which does not have this change (we'll likely have to backport the jortestkit update), but I think it would be better to use release branches instead.